cip/CIP2015-05-13-EXISTS.adoc

```diff
@@ -0,0 +1,167 @@
+= CIP2015-05-13 - EXISTENTIAL SUBQUERIES
+:numbered:
+:toc:
+:toc-placement: macro
+:source-highlighter: codemirror
+
+*Authors:* Andrés Taylor <andres@neotechnology.com>
+
+
+[abstract]
+.Abstract
+--
+This CIP introduces EXISTS, a general function that can test for the existence of properties, simple patterns and full subqueries.
```

Comment:
Can the distinction between when `exists` is a function and when `EXISTS` is a keyword be made explicit somewhere?